independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -108,7 +107,7 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
             final CodestartProjectDefinition projectDefinition = catalog.createProject(input);
             projectDefinition.generate(invocation.getQuarkusProject().getProjectDirPath());
             invocation.log()
-                    .info("\n-----------\n" + MessageIcons.NOOP_ICON + " "
+                    .info("\n-----------\n" + " "
```

Comment:
Why did you remove this icon?, This was badly named but looked great:
![image](https://user-images.githubusercontent.com/2223984/107192748-5094e400-69ee-11eb-8e67-acf2b8392e6b.png)

Reply:
We should add a new icon named `SUCCESS_ICON` with the thumbs up and [SUCCESS] on Windows to use here and everywhere we used the NOOP to mean SUCCESS.

If there are some place `[NOOP]` is used a it should  (I am not sure there is), we should also choose a new unix icon for it.